test/FunctionsV2/EntityMethodDiscoveryTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.Azure.WebJobs;
```

Comment:
Can you add our standard copyright header to the top of this file? StyleCop failed the build because this was missing.

Reply:
Done